static/src/js/middleware/metrics/events.js

```diff
@@ -305,3 +306,18 @@ export const relatedCollection = (action) => {
     relatedCollectionLabel: collectionId
   })
 }
+
+/**
```

Comment:
This makes the rendering time available to Playwright via the console log when EDSC is running in development (i.e. locally and in GitHub Actions).
Later work will handle the other side of the conditional: when EDSC is deployed, send the metric to Google Analytics so it can report on the diversity of real user experiences.